api/v1/db/oracledb/game-dao.js

```diff
@@ -0,0 +1,61 @@
+const appRoot = require('app-root-path');
+const _ = require('lodash');
+
+const { serializeGames } = require('../../serializers/games-serializer');
+
+const conn = appRoot.require('api/v1/db/oracledb/connection');
+
+/**
+ * @summary Return a list of games merged by GAME_ID property.
+ * @param {[Object]} rawGames An array of raw game data returned from SQL database.
+ * @returns {[Object]} Game objects merged.
+ */
+const mergeRawGames = (rawGames) => {
+  const groupedRawGames = _.groupBy(rawGames, 'GAME_ID');
+  const mergedRawGames = _.map(groupedRawGames, (gameMetaDataArray) => {
+    gameMetaDataArray[0].tableCards = gameMetaDataArray[0].CARD_NUMBER === null ? []
+      : _.map(gameMetaDataArray, data => ({
+        cardNumber: data.CARD_NUMBER,
+        cardSuit: data.SUIT,
+      }));
+    return gameMetaDataArray[0];
+  });
+  return mergedRawGames;
+};
```

Comment:
Now that I think about it, `mergeRawGames` should belong in the games serializer. The purpose of the DAO should be just to take information about a query and fetch data from the data source without any serialization or modification. The serializer should handle converting the raw data to the serialized response data. Doing some of the serialization here and the rest in the serializer doesn't seem that intuitive.